### PR TITLE
added additional instructions to contributing.md

### PR DESCRIPTION
**Pull request by cg-cnu**
Added the below instruction from `ISSUE_TEMPLATE.md` to `CONTRIBUTING.md`

` * Please provide a minimal code sample to reproduce the issue if possible.`

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ Before you create a new Issue:
 * Check the [Issues](https://github.com/octokit/rest.js/issues) on Github to ensure one doesn't already exist.
 * Clearly describe the issue, including the steps to reproduce the issue.
 * Please include links to the corresponding github documentation.
+* Please provide a minimal code sample to reproduce the issue if possible.
```
Comment (gr2m) [inline CONTRIBUTING.md]: Could add a 2nd sentence asking for Runkit snippet such as https://runkit.com/gr2m/5aa034f1440b420012a6eebf. It is really _so very helpful_ if people create these

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -13,8 +13,9 @@ By participating in this project you agree to abide by its terms.
 ## Creating an Issue
 
 Before you create a new Issue:
-* Check the [Issues](https://github.com/octokit/rest.js/issues) on Github to ensure one doesn't already exist.
-* Clearly describe the issue, including the steps to reproduce the issue.
+1. Please make sure there is no [open issue](https://github.com/octokit/rest.js/issues?utf8=%E2%9C%93&q=is%3Aissue) yet.
+2. If it is a bug report, include the steps to reproduce the issue and please create a reproducible test case on [runkit.com](https://runkit.com/). Example: https://runkit.com/gr2m/5aa034f1440b420012a6eebf
+3. If it is a feature request, please share the motivation for the new feature and how you would implement it.
 * Please include links to the corresponding github documentation.
```
Comment (gr2m) [inline CONTRIBUTING.md]: it’s now mixing numeral list items with a bullet point. Can you turn the last point to `4. ` please?

Comment (cg-cnu) [reply CONTRIBUTING.md]: Hey @gr2m Sorry for not being attentive. Fixed it in another commit. Thanks!

**Review by gr2m — APPROVED** [review]
cheers :)

**Pull request merged**